app/models/Company.js

```diff
@@ -169,47 +172,45 @@ schema.statics.updateView = async function (id) {
  * @param callback
  */
 schema.methods.comparePassword = async function (password, callback) {
-    console.log("comparePassword ,,,,", password, this.password)
-    // header already sent
-    return await bcrypt.compare(password, this.password);
+  console.log("comparePassword ,,,,", password, this.password);
+  // header already sent
+  return await bcrypt.compare(password, this.password);
 };
 /**
  * Compare raw and encrypted password
  * @param password
  * @param callback
  */
 schema.methods.compareRowPassword = function (password, callback) {
-    if (password === this.password) {
-        callback(null, true);
-    } else {
-        callback(null, false);
-    }
+  if (password === this.password) {
+    callback(null, true);
+  } else {
+    callback(null, false);
+  }
 };
 
-
-schema.virtual('fullpicture').get(function () {
-    let fullpicture = "";
-    if (this.picture.length > 2) {
-        fullpicture = _config("app.imageurl") + this.picture;
-    }
-    return fullpicture;
+schema.virtual("fullpicture").get(function () {
+  let fullpicture = "";
+  if (this.picture.length > 2) {
```

Comment:
I'm not sure about the full functionality of this function, but if no sanitation is implemented, we should limit the type of files that are uploadable and the size of the file (dimensions + size) from the backend to avoid cross-site scripting and files large enough to break the system